reactive-messaging-redisstream-connector/src/main/java/hu/icellmobilsoft/reactive/messaging/redis/streams/RedisStreamsConnector.java

```diff
@@ -81,7 +81,7 @@
 @ConnectorAttribute(name = "xread-block-ms", description = "The milliseconds to wait in an XREADGROUP call", type = "int", defaultValue = "5000",
         direction = ConnectorAttribute.Direction.INCOMING)
 @ConnectorAttribute(name = "xread-noack", description = "Include the NOACK parameter in the XREADGROUP call", type = "boolean",
-        defaultValue = "false",
+        defaultValue = "true",
```

Comment:
majd a doksiban is vezessük át légyszi: https://github.com/i-Cell-Mobilsoft-Open-Source/quarkus-redisstream-messaging/blob/main/docs/connector/configuration-reference.adoc